packages/perseus/src/widgets/passage.tsx

```diff
@@ -512,9 +512,9 @@ export class Passage extends React.Component<PassageProps, PassageState> {
 
         return (
             <div>
-                <div className="perseus-widget-passage-container">
+                <div id="perseus-widget-passage-container">
```

Comment:
🤔 I see how this fixes the issue, but I'm concerned about moving to id's here as we could (maybe theoretically) have more than one passage widget on the page (yielding multiple elements with the same id, which is not allowed).

> The id [global attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes) defines an identifier (ID) which must be unique in the whole document.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
@jeremywiebe Yeah, I thought about that, but I don't think _practically_ that we'd ever encounter it. Plus even if we did, it'd be a warning, not a show-stopping issue.

It doesn't fix the issue for the test prep passages anyway, so this issue is still open. I'll keep exploring.